### PR TITLE
Ratings

### PR DESCRIPTION
**Pull request by sohan9819**
Added the rating input and rating badges in the component library

image

image

### CODE REVIEW

**Review by Asuraking1n — APPROVED** [review]
Liked the way u added rating functionality.
Colors are also good

File: css/components/rating.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+.rating > label {
+  float: right;
+  display: inline;
+  padding: 0;
+  margin: 0;
+  position: relative;
+  width: 1.1em;
+  cursor: pointer;
+  color: #000;
```
Comment (chanchal-panpaliya) [inline css/components/rating.css:13]: use var()

**Review by chanchal-panpaliya — COMMENTED** [review]
The code is cleanly written, with meaningful name .

**Comment by sohan9819** [comment]
@chanchal-panpaliya  thanks for pointing out . Done with the change👍.

**Pull request merged**